### ISSUE TITLE
create_redirect tasks fail due to incorrect primitive type for target URI

### ISSUE THREAD

**Issue by mjordan** [mjordan/islandora_workbench#833]
Given in input CSV like this:

```
redirect_source,redirect_target
aldine-collection/aldine-collection,aldine-collection
```

logging reveals the following information:

```
18-Oct-24 07:05:35 - INFO - {'type': [{'value': 'redirect'}], 'redirect_source': [{'path': 'aldine-collection/aldine-collection'}], 'redirect_redirect': [{'uri': 'aldine-collection'}], 'status_code': [{'value': 301}]}
18-Oct-24 07:05:35 - INFO - {"message":"Unprocessable Entity: validation failed.\nredirect_redirect.0: The path \u0027aldine-collection\u0027 is invalid.\nredirect_redirect.0.uri: This value should be of the correct primitive type.\n"}
18-Oct-24 07:05:35 - ERROR - Redirect from "aldine-collection/aldine-collection" to "aldine-collection" not created. HTTP status code: 422
```

tldr is that the target value, in this case `aldine-collection`, is not of the correct "primitive type".

**Comment by mjordan** [comment]
Resolved with 4358d416942dd5195cd8fda8503b97ef07ada08f.
